barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -115,19 +115,7 @@ std::shared_ptr<typename VerifierInstances::Instance> ProtoGalaxyVerifier_<Verif
     next_accumulator->verification_key->num_public_inputs = accumulator->verification_key->num_public_inputs;
     next_accumulator->public_inputs =
         std::vector<FF>(static_cast<size_t>(next_accumulator->verification_key->num_public_inputs), 0);
-    size_t public_input_idx = 0;
-    for (auto& public_input : next_accumulator->public_inputs) {
-        size_t inst = 0;
-        for (auto& instance : instances) {
-            // TODO(https://github.com/AztecProtocol/barretenberg/issues/830)
-            if (instance->verification_key->num_public_inputs >=
-                next_accumulator->verification_key->num_public_inputs) {
-                public_input += instance->public_inputs[public_input_idx] * lagranges[inst];
-                inst++;
-            }
-        }
-        public_input_idx++;
-    }
+
```

Comment:
There is also no reason to have the line above or, in fact, the public_inputs data structure in prover and verifier instance. Would you please remove them?

Reply:
Also in the protogalaxy recursive verifier

Reply:
I removed a bit more but I'm leaving the pub inputs member in the verifier instance for now since its used in the PG rec verifier for the PI delta. I don't want this to spiral too much. Probably the PG reg verifier should be using oink like the native verifier?

Reply:
Sounds good, thanks for this!